client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -63,6 +63,13 @@ function getHoldMessages( context: string | null, translate: LocalizeProps[ 'tra
 			),
 			supportUrl: null,
 		},
+		SITE_NOT_PUBLIC: {
+			title: translate( 'Make your site public' ),
+			description: translate(
+				'Only you and those you invite can view your site. Make your site visible public.'
```

Comment:
There's a typo here:
`Make your site visible public.`

...should that second sentence be removed?